api/net/Areas/Admin/Models/User/UserModel.cs

```diff
@@ -104,9 +104,7 @@ public UserModel(Entities.User entity) : base(entity)
         this.EmailVerified = entity.EmailVerified;
         this.LastLoginOn = entity.LastLoginOn;
         this.Note = entity.Note;
-
-        this.Roles = entity.RolesManyToMany.Where(r => r.Role != null).Select(r => new RoleModel(r.Role!));
-        this.Roles = entity.Roles.Select(r => new RoleModel(r));
+        this.Roles = entity.Roles.Split(",").Select(r => r[1..^1]);
```

Comment:
We store the roles in the database in the following format `"[administrator],[editor]"`.  We do this so that we have a simple way to search for them.  It's not ideal, but for a quick solution it will work for now.